onchain/rollups/contracts/portals/IEtherPortal.sol

```diff
@@ -26,7 +26,7 @@ interface IEtherPortal is IPortal {
     /// @param _dapp The address of the DApp
     /// @param _execLayerData Additional data to be interpreted by the execution layer
     /// @dev All the value sent through this function is forwarded to the DApp.
-    /// If the transfer fails, an `EtherTransferFailed` error is raised.
+    ///      If the transfer fails, `EtherTransferFailed` error is raised.
```

Comment:
Why remove the indefinite article "an"?

Reply:
Well, that is half true. You could think of error types like classes. A class can be instantiated multiple times. So, you'd say that a function returns an instance, and not the class itself. Likewise, a function may raise an error of a specific type, but not the error type itself.